config/config.go

```diff
@@ -1722,17 +1722,20 @@ type StatefulSetImage struct {
 }
 
 type Image struct {
-	// Registry is the registry of the container image, e.g. my-registry.com or ghcr.io. This setting can be globally
-	// overridden via the controlPlane.advanced.defaultImageRegistry option. Empty means docker hub.
+	// Registry is the registry of the container image reference, e.g. ghcr.io.
 	Registry string `json:"registry,omitempty"`
 
-	// Repository is the repository of the container image, e.g. my-repo/my-image
+	// Repository is the repository of the container image reference, e.g. my-repo/my-image.
 	Repository string `json:"repository,omitempty"`
 
-	// Tag is the tag of the container image, and is the default version.
+	// Tag is the tag of the container image reference, and is the default version.
 	Tag string `json:"tag,omitempty"`
 }
 
+func (i Image) String() string {
+	return fmt.Sprintf("%s/%s:%s", strings.TrimSuffix(i.Registry, "/"), i.Repository, i.Tag)
```

Comment:
What happens if registry or tag is empty? Do we not want to allow this? E.g. "alpine" is a valid docker tag. Since this should be a generic function, we should add this.